pkg/controller/networkpolicy/clustergroup.go

```diff
@@ -393,23 +402,34 @@ func (c *NetworkPolicyController) GetAssociatedGroups(name, namespace string) ([
 
 // getAssociatedGroupsByName retrieves the internal Group and all it's parent Group objects
 // (if any) by Group name.
-func (c *NetworkPolicyController) getAssociatedGroupsByName(grpName string) []antreatypes.Group {
+func (c *NetworkPolicyController) getAssociatedGroupsByName(grpName string) ([]antreatypes.Group, error) {
```

Comment:
not resolved, assuming thumbs up means agreement.

Reply:
I was thinking on solving it in potentially another PR since the error returned by getting parent groups is outside of scope of this PR, and as far as I remember there are other places throwing the same errors as well. But I have refactored all group related ones.